src/App.js

```diff
@@ -1,5 +1,133 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+
 class App {
-  play() {}
+  play() {
+    MissionUtils.Console.print("숫자 야구 게임을 시작합니다.");
+    const computer = this.computerInput();
+    this.gameManager(computer);
+  }
+
+  computerInput() {
+    const computer = [];
+    while (computer.length < 3) {
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if (!computer.includes(number))
+        computer.push(number);
+    }
+    return computer;
+  }
+
+  gameManager(computer) {
+    MissionUtils.Console.readLine('숫자를 입력해주세요 : ', (player) => {
+      this.playerCheck(player);
+      let [strikeCount, ballCount] = this.checkBallCounts(computer, player.split("").map(Number));
+      this.printBallCounts(strikeCount, ballCount);
+      
+      if(!this.isClear(strikeCount))
+        this.gameManager(computer);
+      else
+        this.retry();
+    });
+  }
+
+  retry() {
+    MissionUtils.Console.readLine('게임을 새로 시작하려면 1, 종료하려면 2를 입력하세요.', (number) => {
+      this.retryCheck(number);
+      if(number === "1")
+        this.play();
+      else {
+        MissionUtils.Console.close();
+        return 0;
+      }
```

Comment:
`else`문 안에서는 `return`을 지양하는 것이 좋을 것 같아요! 다음과 같은 컨벤션도 있답니다!

https://github.com/airbnb/javascript#arrays--callback-return

대신 이렇게 고쳐보면 어떨까요?
```suggestion
      if(number === "1")
        return this.play();
      return MissionUtils.Console.close();
```